src/MusicTheory/Pitch/Enharmonic.elm

```diff
@@ -0,0 +1,9 @@
+module MusicTheory.Pitch.Enharmonic exposing (simple)
+
+import MusicTheory.Pitch as Pitch exposing (Pitch)
+import MusicTheory.PitchClass.Enharmonic as PitchClassEnharmonic
+
+
+simple : Pitch -> Pitch
+simple pitch =
+    pitch |> Pitch.map PitchClassEnharmonic.simple
```

Comment:
This implementation ignores the `Octave` of the `Pitch`. So:
```
Pitch (PitchClass C Flat) Four
|> simple
```
would equal `Pitch (PitchClass B Natural) Four`.

But this is incorrect, since Cb4 == B3.